lldb/docs/python_extensions.rst

```diff
@@ -0,0 +1,36 @@
+Python Extensions
+=================
+
+LLDB provides many scriptable extensions to augment the debugger capabilities
+and give the ability to the user to tailor their experience to their own needs.
```

Comment:
Suggestion: `LLDB provides scriptable extensions to augment the debugger's capabilities. This gives users the ability to tailor their debugging experience to their own needs.`